packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,145 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+
+/* 
+ Styles that are specific to VA.gov using CSS type selectors (elements/tags).
+*/
+
+.feature {
+  background: $color-primary-alt-lightest;
+  padding: 1em;
+  clear: both;
+  margin: 1.5em 0 1.5em 0;
+
+  p {
+    color: inherit;
+    font: inherit;
+    margin-bottom: 0;
+    margin-top: 0;
+    padding-bottom: 0.5em;
+  }
+
+  h2 {
+    margin-top: 0;
+  }
+
+  h3 {
+    margin-top: 0;
+  }
+  h4 {
+    margin-top: 0;
+  }
+  ul {
+    margin: 0 0 0.5em 1.5em;
+    padding: 0;
+    li {
+      list-style: square;
+      margin: 0;
+    }
+  }
+}
+
+.va-quicklinks {
+  ul {
+    margin-top: 12.8px; // Original 0.8rem to 16px base font
+  }
+  li {
+    padding: 8.96px 0; // Original 0.56rem to 16px base font
+  }
+  h3 {
+    line-height: 1.2em;
+    font-size: 1.65em;
+    color: $color-primary-darker;
+    margin: 0 0 16px 0; // Original 1rem to 16px base font
+    padding: 0 0 0.2em 0;
+  }
+}
+
+// Playbook and Facility Locator
+// Used in content-build: src/site/includes/breadcrumbs-playbook.html
+
+.va-nav-breadcrumbs--playbook {
```

Comment:
This is also only being used in one place in the content-build. Is it possible to move this somewhere there? If so, we don't need this to be loaded globally and could remove it from here:

<img width="583" alt="Screenshot 2023-12-11 at 3 35 10 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/a77a4bf8-d8e5-4052-b4b5-90df1cfa81eb">

Reply:
Yes, this instance and the one mentioned above are definitely worth checking to see if they can just be brought into content-build.